src/ext.ts

```diff
@@ -1,5 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
what does this mean?

Reply:
as qlik-modifiers does not contain types, it is a workaround to let typescript not complain.
I saw that people did the same for sn-table.
<img width="510" alt="Screenshot 2022-06-29 at 22 24 57" src="https://user-images.githubusercontent.com/4471489/176538104-09c743d3-a6f8-4216-ada4-284f4bc4912b.png">

Reply:
ok. If we notice that we need  // @ts-ignore in other places we can disable the `@typescript-eslint/ban-ts-comment` rule